docs/schema-loader.md

```diff
@@ -182,139 +219,184 @@ Create/Delete JDBC schemas
 
```

Comment:
Should we should keep these `Deprecated` parts for these databases in the supported version branches (`3.10` to `3.5`) and on the `master` branch? Or we should completely remove these from the doc in all or some branches?

> **Note**
> 
> I haven't revised these parts yet because I wanted to confirm if we need to keep them first.

Reply:
I leave it to @brfrn169 and @feeblefakie to decide whether we should keep them. If we keep it, I think it would be good if we could hide them within the collapsible box.

Reply:
Since the deprecated commands will be removed in `5.0`, we need to keep the `Deprecated` parts.

Reply:
> I think using tabs here and hiding the content unless the user clicks on one of them might work
I agree that this would be helpful for users, too. Thanks!

Reply:
@choplin After some testing, it looks like we can't hide the contents of a tab. In the following screenshot, when the user selects `Cassandra`, the content doesn't appear as expected (other tabs work fine though). Also, even if we could, the tabbed section looks a little awkward without any content showing:

<img width="960" alt="tabbed-content-initial-view-disabled" src="https://github.com/scalar-labs/scalardb/assets/23216828/4256c737-bbc3-4c99-82c9-28cbcd9fac3c">

For now, I've implemented the following style that uses tabs in a **Note** block:

<img width="960" alt="tabbed-content-initial-view-enabled-in-note-block" src="https://github.com/scalar-labs/scalardb/assets/23216828/1262a3df-6bb8-4549-8a11-250d6e2f65ce">

Do you have any concerns about this method of showing deprecated content?